website/pages/en/sunrise.mdx

```diff
@@ -12,15 +12,15 @@ This plan draws on many previous developments from The Graph ecosystem, includin
 
 ### What are the phases of the Sunrise?
 
-**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network as well as simple payment options.  
+**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.  
```

Comment:
Is the last item in the enumeration missing a verb?

Reply:
offers!